index.html

```diff
@@ -53,9 +87,67 @@ <h3 class="multipost-h">Multi-Post Stories</h3>
                 <li class="rubox">Ruby</li>
             </ul>
             <button class="button1" type="button">See Project</button>
+
         </div>
-    </section>
+        </section>
+    
+            <!-- For media css -->
+    <section class="mediacss-grid">
+        <div class="gitem1">
+            <button class="buttongrid1" type="button">See Project</button>
+        </div>
+        <div class="gitem2">
+            <h4 class="gitem2-h">Data Dashboard Healthcare</h4>
+            <p class="gitem2-p">A daily selection of privately personalized
+            reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <ul class="gitem2-s">
+                <li>html</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
+            </ul>
+        </div>
+        <div class="gitem3">
+            <h4 class="gitem3-h">Website Protfolio</h4>
+            <p class="gitem3-p">A daily selection of privately personalized
+            reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <ul class="gitem3-s">
+                <li>html</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
+            </ul>
+        </div>
```

Comment:
## 1.
### Your Project
![center-projects (copy)](https://user-images.githubusercontent.com/93039359/177851532-4843401c-e4ad-4aec-8e42-0d4f7487af1d.png)

### Figma Template
![Figma-all-projects](https://user-images.githubusercontent.com/93039359/177851550-70d3dce4-e6d3-4085-9a8e-cb247fa14171.png)

- [ ] Green box shows the correct position of the project container. Please follow the Figma template. :blush:

## 2.

### Your Project
![center-projects (another copy)](https://user-images.githubusercontent.com/93039359/177853116-f01d274d-f0af-47d9-ab7f-fecfd4b0209d.png)


### Figma Template
![Figma-all-projects](https://user-images.githubusercontent.com/93039359/177853247-a890cd9b-0aed-468b-9521-7c06b3e29283.png)


- [ ] If you would make background-size: cover; . Please follow the Figma template. :blush:
#### **Solution for your background**